doc/build-osx.md

```diff
@@ -51,6 +51,18 @@ To install, run the following from your terminal:
 brew install automake libtool boost pkg-config libevent
 ```
 
+For macOS 11 (Big Sur) and 12 (Monterey) you need to install a more recent version of llvm.
```

Comment:
That's not that long ago though, and we have `OSX_MIN_VERSION=11.0` for now.

Reply:
> That's not that long ago though, and we have `OSX_MIN_VERSION=11.0` for now.

Just because we build release binaries that support that operating system, does not mean you need to be able to compile on that system (regardless of using the system compiler).

Reply:
That's also true, though so far I haven't seen evidence that compilation is impossible.

Homebrews main motivation for dropping macOS 11 seems to be support requests. From their install script:

> This installation may not succeed.
After installation, you will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew\'s GitHub,
Twitter or any other official channels. You are responsible for resolving any
issues you experience while you are running this

So far I haven't seen a report of anyone even _trying_ to compile on that version recently, let alone feel overwhelmed by their support requests :-)